src/client/pages/ConsentsCommunication.tsx

```diff
@@ -1,53 +1,99 @@
 import React from 'react';
 import { css } from '@emotion/react';
-import { from, space } from '@guardian/source-foundations';
+import {
+  from,
+  space,
+  brand,
+  neutral,
+  headline,
+} from '@guardian/source-foundations';
 import {
   getAutoRow,
   gridItemColumnConsents,
   consentsParagraphSpanDef,
 } from '@/client/styles/Grid';
-import { CommunicationCard } from '@/client/components/CommunicationCard';
```

Comment:
I don't believe there are any other usages of `CommunicationCard`, could be cleaned up perhaps.